validation/jinja2/validation/list_phrases.html

```diff
@@ -21,26 +21,47 @@
 {% import 'validation/_segment_card.html' as segment %}
 
 {% block content %}
-<div>
-  <form class="form-inline ml-2 my-2 my-lg-0" method="GET" action="{{
+<div class="center">
+  <form method="GET" action="{{
     url('validation:index')
   }}">
     <div class="filter">
-      <button class="{{ all_class }}" type="submit" id="filter-all" name="mode" value="all">All</button>
-      <button class="{{ new_class }}" type="submit" id="filter-new" name="mode" value="new">New</button>
-      <button class="{{ linked_class }}" type="submit" id="filter-linked" name="mode" value="linked">Linked</button>
+        Filter by:
+        <input type="radio" id="filter-all" name="mode" value="all">
+        <label for="all"> All</label>
```

Comment:
I think the `for` is supposed to match `id`, i.e., `for="filter-all"`. Does clicking the label select the radio button, or do you have to click the little circle of the radio button itself?

Reply:
You are correct! Clicking the label didn't work before, but after changing this, you can click the label or directly on the little circle.